components/email-mgt/org.wso2.carbon.email.mgt/src/main/java/org/wso2/carbon/email/mgt/EmailTemplateManagerImpl.java

```diff
@@ -422,24 +422,10 @@ public void deleteEmailTemplates(String templateTypeName, String tenantDomain, S
     }
 
     @Override
+    @Deprecated
     public void addDefaultEmailTemplates(String tenantDomain) throws I18nEmailMgtException {
 
-        try {
```

Comment:
Thinking again, throwing an exception in the deprecated method is not good.. Lets keep the current impl as it is and add a warn log..

Reply:
Addressed with https://github.com/wso2-extensions/identity-event-handler-notification/pull/258/commits/2b9424930522b1efcfb51478d134f925b6393352